sdk/tests/sdk/whirlpools/quote/increase-liquidity-quote.test.ts

```diff
@@ -1,10 +1,226 @@
+        isTokenA,
+        slippage
+      }));
+
+      it("[|---|---P------]", async () => testVariation({
+        pTickLowerIndex: -202,
+        pTickUpperIndex: 555,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+
+      it("[--|---|--P-------]", async () => testVariation({
+        pTickLowerIndex: -100,
+        pTickUpperIndex: 555,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+    });
+
+    describe("Current Price in range", () => {
+      it("|-----[---P---]-----|", async () => testVariation({
+        pTickLowerIndex: -100,
+        pTickUpperIndex: 100,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+
+      it("[--|----P----]-----|", async () => testVariation({
+        pTickLowerIndex: -300,
+        pTickUpperIndex: 100,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+
+      it("|--[---P---|-----]", async () => testVariation({
+        pTickLowerIndex: -100,
+        pTickUpperIndex: 300,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+
+      it("[---|---P---|----]", async () => testVariation({
+        pTickLowerIndex: -300,
+        pTickUpperIndex: 300,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+    });
+
+    describe("Current Price below range", () => {
+      it("[---P---] |---------|", async () => testVariation({
+        pTickLowerIndex: -500,
+        pTickUpperIndex: -300,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+
+      it("[---P--|---]------|", async () => testVariation({
+        pTickLowerIndex: -500,
+        pTickUpperIndex: -100,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+
+      it("[-----P--|---|]", async () => testVariation({
+        pTickLowerIndex: -500,
+        pTickUpperIndex: 199,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+
+      it("[-------P--|---|--]", async () => testVariation({
+        pTickLowerIndex: -500,
+        pTickUpperIndex: 300,
+        tickCurrentIndex: 0,
+        inputTokenAmount: new BN(100000),
+        isTokenA,
+        slippage
+      }));
+    });
+
+    function testVariation(params: { pTickLowerIndex: number, pTickUpperIndex: number, tickCurrentIndex: number, inputTokenAmount: BN, isTokenA: boolean, slippage: Percentage }) {
+      const { pTickLowerIndex, pTickUpperIndex, tickCurrentIndex, inputTokenAmount, isTokenA } = params;
+      const slippage = Percentage.fromFraction(1, 100);
```

Comment:
Does this override `slippage` param, and always 1% slippage is used ?

Reply:
removed!